database/src/migrations/release.0.5/restoration.sql

```diff
@@ -1134,8 +1134,8 @@ CREATE TABLE project_spatial_component(
     project_spatial_component_type_id    integer                     NOT NULL,
     name                                 varchar(50)                 NOT NULL,
     description                          varchar(3000),
-    priority                             character(1)                DEFAULT 'N' NOT NULL,
     geometry                             geometry(geometry, 3005),
+    priority                             character(1)                NOT NULL,
```

Comment:
Just checking: Did we still want the `DEFAULT 'N'` property?

Reply:
i would say yes. for logical flags you have two types... one with Y/N and the other with Y/N/Unknown (ie. null). the expectation is that the app will always set this but we need a default value to satisfy not having an "unknown" value.

Reply:
Should we just modify this PR directly to add it back, or is it better if you do it on your end through your tool?

Reply:
please find this pushed through...